tests/publiccloud/run_ltp.pm

```diff
@@ -147,10 +147,15 @@ sub run {
     assert_script_run($log_start_cmd);
 
     # LTP command line preparation
+    assert_script_run('rpm -q python3-paramiko; true');
+    assert_script_run('zypper -n se -v paramiko; true');
     zypper_call("in -y python3-paramiko python3-scp");
```

Comment:
Why?

Reply:
I don't know.

Reply:
It now has a purpose. I commented it. Please re-check.